code/game/objects/structures/statues.dm

```diff
@@ -232,6 +247,7 @@
 
 /obj/structure/statue/bananium/clown
 	name = "statue of a clown"
+	desc = "A bananium statue of a clown. Somehow just the sight of it infuriates you."
```

Comment:
`Somehow, just` would look better here imo. It just feels like it should have a pause after it.